### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by dmlap** [comment]
Test?

File: test/playlist-maker-test.js [hunk 1]
```diff
@@ -198,3 +198,33 @@ q.test('when loading a new playlist, trigger "playlistchange" on the player', fu
...
   window.setTimeout = oldTimeout;
 });
+
+q.test('cleartimeout on dispose', function() {
+  var oldTimeout = window.setTimeout;
+  var oldClear = window.clearTimeout;
+  var Player = function(proxy) {
+    extend(true, this, proxy);
+  };
+  Player.prototype = Object.create(playerProxy);
```
Comment (dmlap) [inline test/playlist-maker-test.js]: Wow, fancy stuff.

Comment (gkatsev) [inline test/playlist-maker-test.js]: unfortunately, it means these tests won't run in IE8, but not a huge loss.

Comment (dmlap) [inline test/playlist-maker-test.js]: You're lucky @heff isn't reviewing this PR

**Comment by dmlap** [comment]
lgtm

File: test/playlist-maker-test.js (hunk 1 above)
Comment (gkatsev) [inline test/playlist-maker-test.js]: :D
It's possible to make it run on IE8, it just doesn't currently.

**Pull request merged**